Composer/Dockerfile

```diff
@@ -18,8 +18,236 @@ ENV NODE_ENV "production"
 RUN yarn build:prod
 
 
-# use a multi-stage build to reduce the final image size
-FROM node:12-alpine
+# Define arg(s) needed for the From statement
+FROM alpine:3.8 as installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.2.0
+ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-alpine-x64.tar.gz
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+ARG PS_INSTALL_VERSION=6
+
+# Download the Linux tar.gz and save it
+ADD ${PS_PACKAGE_URL} /tmp/linux.tar.gz
+
+# define the folder we will be installing PowerShell to
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION
```

Comment:
Why is powershell a system requirement? Do mac users also need to have it installed?

Reply:
In order to run powershell script (such like create.ps1, deploy.ps1 before, and I add a build script to ensure user's dotnet version can run our csharp runtime). 
I use node image as base image and use a simpler way to install pwsh.